client/src/common/components/ActionList/ActionItems/ActionIconItem.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {ViewStyle} from 'react-native';
+import styled from 'styled-components/native';
+import {SPACINGS} from '../../../constants/spacings';
+import {IconType} from '../../Icons';
+import {Spacer4} from '../../Spacers/Spacer';
+import ActionItem from '../ActionItem';
+
+const ActionWrapper = styled(ActionItem)({
+  paddingVertical: SPACINGS.EIGHT,
+  paddingHorizontal: SPACINGS.EIGHT,
+});
+
+const IconWrapper = styled.View({
+  width: 30,
+  height: 30,
+});
+
+type ActionIconItemProps = {
+  Icon?: IconType;
+  style?: ViewStyle;
+  children: React.ReactNode;
+};
+const ActionIconItem: React.FC<ActionIconItemProps> = ({
+  Icon,
+  style,
+  children,
+}) => (
+  <ActionWrapper style={style}>
+    {Icon && (
```

Comment:
`ActionIconItem` is named so you assume it takes an Icon, not an optional Icon. Is it going to be used without Icon?

Reply:
Hehe true... lack of better naming maybe. I'll figure something out 🤔

Reply:
Bleh, can't come up with anyting better tbh :D